### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by ChrisMaddock**
Hopefully this is the correct fix?

### CODE REVIEW

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ To enable this feature, an application needs to target .NET Framework 4.8 or opt
 Once enabled, all controls that contain a tooltip display it once the control receives keyboard focus. The tooltip can be dismissed over time or when the keyboard focus changes. Users can also dismiss the tooltip manually by using a new keyboard shortcut, Ctrl + Shift + F10. Once the tooltip has been dismissed it can be displayed again by using the same keyboard shortcut.
 
 > [!NOTE]
-> [Ribbon tooltips](xref:System.Windows.Controls.Ribbon.RibbonToolTips> on <xref:System.Windows.Controls.Ribbon.Ribbon> controls won’t show on keyboard focus; they only show via the keyboard shortcut.
+> <xref:System.Windows.Controls.Ribbon.RibbonToolTips> on <xref:System.Windows.Controls.Ribbon.Ribbon> controls won’t show on keyboard focus; they only show via the keyboard shortcut.
```
Comment (rpetrusha) [inline docs/framework/whats-new/whats-new-in-accessibility.md]:
```suggestion
> [Ribbon tooltips](xref:System.Windows.Controls.Ribbon.RibbonToolTips) on <xref:System.Windows.Controls.Ribbon.Ribbon> controls won’t show on keyboard focus; they only show via the keyboard shortcut.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for noticing the badly formatted link, @ChrisMaddock. A better fix, though, is to retain the original link text but correct the link.

File: docs/framework/whats-new/whats-new-in-accessibility.md (hunk 1 above)
Comment (ChrisMaddock) [reply docs/framework/whats-new/whats-new-in-accessibility.md]: Thanks!

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional change, @ChrisMaddock. I'll merge your PR now.

**Pull request merged**